internal/providers/system/system.go

```diff
@@ -56,3 +71,31 @@ func fetchConfig(logger *log.Logger, filename string) (types.Config, report.Repo
 	}
 	return util.ParseConfig(logger, rawConfig)
 }
+
+// fetchBaseConfig is a helper function to merge all the base config fragments inside of a particular directory.
+func fetchBaseConfig(logger *log.Logger, dir string) (types.Config, report.Report, error) {
```

Comment:
Can we put "directory" into the function name?  The names of `FetchBaseConfig` and `fetchBaseConfig` don't make it clear what the distinction is.